Initial support for FidoNet NodeList

dolikdev opened the pull request:
This PR adds initial support for NodeList. 

New modal window opens by `Tab` key pressed in either *To Name* or *To Address* fields of a new message compose window.
Selected name and address populated into the corresponding fields.
NodeList window may be closed by `Esc` key and will not change original fields content in this case.
Sysop Name, BBS name and City are displayed as in the NodeList without any changes. 
- Question: Shall we consider replacement of `_` symbol with a white-space?

Further improvements may include: default position to self-address or net, search by name and address, replacement for City.yaml, point-list and private address book support.

dolikdev commented:
The case statement CodeFactor complains about was made on purpose as a placeholder for the future search/jump implementation.

askovpen commented:
without nodelist:
`Insert -> Enter -> Enter -> Tab -> Enter`
have crash:
 panic: runtime error: index out of range [0] with length 0
pkg/ui/editheader.go:155 +0x16d

askovpen commented:
`Insert -> Enter -> Enter -> Tab -> Escape`
move to subject?

dolikdev commented:
> `Insert -> Enter -> Enter -> Tab -> Escape` move to subject?

I've changed logic to jump to Subject field only on `Enter`, i.e. address picked from the Nodelist.

> without nodelist: `Insert -> Enter -> Enter -> Tab -> Enter` have crash: panic: runtime error: index out of range [0] with length 0 pkg/ui/editheader.go:155 +0x16d

Fixed.
